R/spark_compile.R

```diff
@@ -126,7 +126,7 @@ spark_compile <- function(name,
 #'
 #' @keywords internal
 #' @export
-spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
+compile_package_jars <- function(package = rprojroot::find_package_root_file(),
                                        spark_versions = NULL,
```

Comment:
Okay, let's leave the scala compiler alone and we'll just call the ones we
know need to be called (it can be a vectorized parameter so users can pass
multiple compilers).

On Fri, Aug 5, 2016 at 9:55 PM, Kevin Ushey <notifications@github.com>
wrote:

> In R/spark_compile.R
> <https://github.com/rstudio/sparklyr/pull/150#discussion_r73778722>:
>
> > @@ -126,7 +126,7 @@ spark_compile <- function(name,
> >  #'
> >  #' @keywords internal
> >  #' @export
> > -spark_compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> > +compile_package_jars <- function(package = rprojroot::find_package_root_file(),
> >                                         spark_versions = NULL,
>
> The tough part about scala_versions is that we don't really have a good
> way of indexing the scala compilers. Maybe we need an entry like:
>
> scala_compilers = list(
>     "2.10" = "path/to/scala-2.10",
>     "2.11" = "path/to/scala-2.11"
> )
>
> or something to that effect?
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/rstudio/sparklyr/pull/150/files/dafaaa2516e9a03fac9075f1222cb9285069e69a..26772d833cb691b37a37157f0c62a54a961be082#r73778722>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAGXx7DKaTx1Gwr7NMztlqFTm-tF1oL4ks5qc-mPgaJpZM4JeE4J>
> .
>